test/unit/test.tour.js

```diff
@@ -7,20 +7,25 @@ import tippy from 'tippy.js';
 import { defaults as tooltipDefaults } from '../../src/js/utils/tooltip-defaults';
 import { spy } from 'sinon';
 
-
 // since importing non UMD, needs assignment
 window.Shepherd = Shepherd;
 
-describe('Tour', function() {
+const DEFAULT_STEP_CLASS = 'shepherd-step-tooltip';
+
+describe('Tour | Top-Level Class', function() {
   let instance, shouldShowStep;
 
   const defaultStepOptions = {
-    classes: 'shepherd-theme-arrows',
+    classes: DEFAULT_STEP_CLASS,
     scrollTo: true
   };
 
+  beforeEach(() => {
+    tippy.disableAnimations();
+  });
+
   afterEach(() => {
-    instance.cancel();
+    instance.complete();
```

Comment:
Is there a reason we need this to be `complete` and not `cancel`? I think both should call `done` right?

Reply:
They do, but cancel has a lot of functionality around displaying promps if the user has that set as an option. So `complete` felt better both semantically  _and_ to reduce the risk of unforseen side-effects.

Reply:
We should be making sure to not be setting `confirmCancel` to true, and this should be a non-issue.

Reply:
Oh, no, it's not an issue now. I'm just thinking it reduces risk of coupling unexpected behavior in the future.

Reply:
Sure, just feels a bit arbitrary to me, but this is fine 👍